src/main/java/org/kainos/ea/cli/JobRoles.java

```diff
@@ -0,0 +1,43 @@
+package org.kainos.ea.cli;
+
+public class JobRoles {
```

Comment:
Probably not best to have this called `JobRoles` when it's only 1 job role

Reply:
I believe this was a previous name somewhere else, so I just followed